metadata/src/main/java/com/redhat/lightblue/metadata/Indexes.java

```diff
@@ -61,6 +63,20 @@ public boolean isEmpty() {
         return indexes.isEmpty();
     }
 
+    public boolean isCaseInsensitiveKey(Path path) {
```

Comment:
This is another one only used in Translator. It's just used as a convenience in case insensitive regex queries to see if we have a case insensitive field available to query.  Since it's only used there at this point, we could move it.